doc/sphinx/source/frameworks/generic.rst

```diff
@@ -89,7 +89,10 @@ Memory allocation
 -----------------
 
 Buffer pointers should be aligned at the 64-byte boundary. NUMA policy should be 
-configured for local memory allocation (``numactl --localloc``)
+configured for local memory allocation (``numactl --localloc``).  For detail on 
+using the 
```

Comment:
This sentence kind of ends in the

Reply:
Can't remember where that was supposed to go.  A link to some Intel doc, probably.. ? Will delete until I can remember / fix.